data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -241,8 +242,8 @@ UPDATE `creature_template` SET `mindmg` = 158, `maxdmg` = 210, `DamageModifier`
 UPDATE `creature_template` SET `mindmg` = 235, `maxdmg` = 312, `DamageModifier` = 1 WHERE `entry` = 7206;
 -- Ironaya 
 UPDATE `creature_template` SET `mindmg` = 354, `maxdmg` = 470, `DamageModifier` = 1 WHERE `entry` = 7228;
--- Galgann Firehammer (Not listed in Beastiary) Src: https://classicdb.ch/?npc=7291
-UPDATE `creature_template` SET `mindmg` = 319, `maxdmg` = 410 WHERE `entry` = 7291;
+-- Galgann Firehammer (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=7291 / https://db.rising-gods.de/?npc=7291
+UPDATE `creature_template` SET `mindmg` = 125, `maxdmg` = 166, `DamageModifier` = 1 WHERE `entry` = 7291;
```

Comment:
this one has quite low damage compared to his the other bosses in the instance, no?